pyquil/tests/test_quil.py

```diff
@@ -398,3 +398,12 @@ def test_prog_merge():
     prog_0 = Program(X(0))
     prog_1 = Program(Y(0))
     assert merge_programs([prog_0, prog_1]).out() == (prog_0 + prog_1).out()
+
+
+def test_get_qubits():
```

Comment:
I think there should be a test that uses qubit allocation. Something along the lines of

```
p = pq.Program()
q = p.alloc()
p.inst(H(q))
p.get_qubits()
```